templates/goaccess.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>goaccess</Name>
+  <Repository>allinurl/goaccess</Repository>
+  <Registry>https://hub.docker.com/r/allinurl/goaccess/</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://github.com/allinurl/goaccess/issues/</Support>
+  <Project>https://goaccess.io/</Project>
+  <Overview>GoAccess is a real-time web log analyzer and interactive viewer that runs in a terminal in *nix systems or through your browser.</Overview>
+  <WebUI>http://[IP]:[PORT:7890]</WebUI>
+  <PostArgs>/var/log/access.log --log-format=COMBINED -o /var/www/goaccess/index.html --real-time-html --port=7890</PostArgs>
+  <Config Name="Log Location" Target="/var/log" Default="" Mode="ro" Description="Access Log Location" Type="Path" Display="always" Required="true" Mask="false"></Config>
```

Comment:
```suggestion
  <Config Name="Log Location" Target="/var/log" Default="" Mode="ro" Description="Location for an access.log file" Type="Path" Display="always" Required="true" Mask="false"></Config>
```

Reply:
wouldnt let me commit this one. I committed separately.